bundles/org.openhab.binding.daikin/src/main/java/org/openhab/binding/daikin/internal/api/airbase/AirbaseEnums.java

```diff
@@ -114,7 +114,7 @@ public static AirbaseFanSpeed fromValue(int rate, boolean auto, boolean airside)
                     return m;
                 }
             }
-            throw new IllegalArgumentException(String.format("Unexpected FanSpeed value from rate=%d, auto=%d, airside=%d", rate, auto?1:0, airside?1:0));
+            LOGGER.warn("Unexpected FanSpeed value from rate={}, auto={}, airside={}", rate, auto?1:0, airside?1:0);
```

Comment:
The problem is: You either need to rteurn a value or throw an exception. If you throw an exception, you need to catch it somewhere.

Reply:
Understood. I have more changes to commit after testing is done.